### DISCUSSION TITLE
Multiple Extensions (Extras) in one text field. Is a divider present?

### DISCUSSION THREAD

**Discussion by Capekk** [gopro/labs#1174] [Q&A] [answered]
Hello everyone,

It's my first time using GoPro Labs and specifically the QR Control app. I'm setting up some QRs for the upcoming skiing season and have two requirements within the "Extras" tab.

It's probably a simple question, however I have no idea how it works so here I am! :)

I want to input, at this time, two different "Extras" commands into my QR code - "EXPT" and "BITR". But I don't know how to input them into one code line. What I'm asking is: Is there any devider like there is in normal coding? (e.g. " ; " or anything else?) I couldn't find it anywhere online and there is no "error" checker (from what I understand). I find it hard to belive that the commands are right next to each other without any spacing or division. 

Right now the code looks like this:

oMEXPT=2oMBITR=180

the commands in use:
1. oMEXPT=2
2. oMBITR=180

Is this the right way to do it or am I missing anything. It's not just for two commands, I'm planning to utilize and layer multiple of them into one QR code in the future.

Thanks for any answers and advice!

**Comment by dnewman-gpsw** [comment, marked as answer]
There are no dividers, QR code benefit from efficient, so no waste on spaces or semicolons. Old format oMEXPT=2oMBITR=180, new format (2024 FW is more effecient) $EXPT=2$BITR=180.  Within QRControl you can enable on the About page
![image](https://github.com/user-attachments/assets/c430931f-0d21-45a4-9b1e-5123238cd436)
